js/tab.js

```diff
@@ -1,5 +1,21 @@
-function openTab(evt, cityName){
-    var i, tabcontent, tablinks;
+function manageHash(defaultID){
+    tablinks = document.getElementsByClassName("tablinks");
+    for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+    }
+    let locationId = window.location.hash.substring(1)
+    locationId = locationId === ''?defaultID:locationId
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+    }
+    document.getElementById(locationId).style.display = "block";
+    document.getElementById(locationId+"Link").classList.add("active")
+}
```

Comment:
Maybe this is obvious to a Javascript/DOM expert (or even to a noob - I don't know), but it is completely mysterious to me. What is this function doing in words? Perhaps add a comment to document the function? Is the name descriptive, i.e. is this function managing some hash?

Reply:
Added comments.